### PR TITLE
add destroy for project

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn ready for review

**Comment by miltontony** [comment]
@smn please review this after the fact

File: unicoremc/models.py [hunk 1]
```diff
@@ -289,3 +290,10 @@ def create_db(self):
 
     def init_db(self):
         self.db_manager.init_db(self.app_type, self.country)
+
+    def destroy(self):
+        shutil.rmtree(self.repo_path())
+        shutil.rmtree(self.frontend_repo_path())
```
Comment (smn) [inline unicoremc/models.py:296]: Something in me expected these to happen in the opposite order.

**Comment by smn** [comment]
:+1:

**Pull request merged**